hail/python/hail/docs/guides/genetics.rst

```diff
@@ -165,6 +165,140 @@ Pruning Variants in Linkage Disequilibrium
             dataset, rather than on the biallelic dataset, so that the biallelic dataset
             does not need to be recomputed.
 
+Analysis
+~~~~~~~~
+
+Linear Regression
+.................
+
+Single Phenotype
+++++++++++++++++
+
+:**tags**: Linear Regression
+
+:**description**: Compute linear regression statistics for a single phenotype.
+
+:**code**:
+
+    Approach #1: Use the :func:`.linear_regression` method
+
+    >>> mt_linreg = hl.linear_regression(y=mt.pheno.height, x=[1, mt.GT.n_alt_alleles()])
+
+    Approach #2: Use the :func:`.aggregators.linreg` aggregator
+
+    >>> mt_linreg = mt.annotate_rows(linreg=hl.agg.linreg(mt.pheno.height, [1, mt.GT.n_alt_alleles()]))
+
+:**dependencies**: :func:`.linear_regression`, :func:`.aggregators.linreg`
+
+:**understanding**:
+
+    .. container:: toggle
+
+        .. container:: toggle-content
+
+            The :func:`.linear_regression` method is more efficient than using the :func:`.aggregators.linreg`
+            aggregator. However, the :func:`.aggregators.linreg` aggregator is more flexible (multiple covariates
+            can vary by entry) and returns a richer set of statistics.
+
+
+Multiple Phenotypes
++++++++++++++++++++
+
+:**tags**: Linear Regression
+
+:**description**: Compute linear regression statistics for multiple phenotypes.
+
+:**code**:
+
+    Approach #1: Use the :func:`.linear_regression` method for all phenotypes simulatenously
+
+    >>> mt_linreg = hl.linear_regression(y=[mt.pheno.height, mt.pheno.blood_pressure],
+    ...                                  x=[1, mt.GT.n_alt_alleles()])
+
+    Approach #2: Use the :func:`.linear_regression` method for each phenotype sequentially
+
+    >>> mt_linreg = hl.linear_regression(y=mt.pheno.height, x=[1, mt.GT.n_alt_alleles()])
+    >>> mt_linreg = hl.linear_regression(y=mt_linreg.pheno.blood_pressure,
+    ...                                  x=[1, mt_linreg.GT.n_alt_alleles()])
+
+    Approach #3: Use the :func:`.aggregators.linreg` aggregator
+
+    >>> mt_linreg = mt.annotate_rows(linreg_height=hl.agg.linreg(mt.pheno.height, [1, mt.GT.n_alt_alleles()]),
+    ...                              linreg_bp=hl.agg.linreg(mt.pheno.blood_pressure, [1, mt.GT.n_alt_alleles()]))
+
+:**dependencies**: :func:`.linear_regression`, :func:`.aggregators.linreg`
+
+:**understanding**:
+
+        .. container:: toggle
+
+        .. container:: toggle-content
+
+            The :func:`.linear_regression` method is more efficient than using the :func:`.aggregators.linreg`
+            aggregator, especially when analyzing many phenotypes. However, the :func:`.aggregators.linreg`
+            aggregator is more flexible (multiple covariates can vary by entry) and returns a richer set of
+            statistics. If the phenotypes being analyzed have different patterns of missingness, you should
+            **not** use the :func:`.linear_regression` method for all phenotypes simulatenously (Approach #1).
```

Comment:
I think this is too strong a statement. In some cases, restricting to the common samples may be reasonable. Instead, just make users aware of the behavior that will result in Approach #1.